docs/storage/volume_migration.md

```diff
@@ -201,6 +201,31 @@ spec:
         pod: {}
 ```
 
+## Volume migration cancellation
+
+The users could, for various reasons, wish to stop and cancel the ongoing volume migration.
+Migration cancellations are also handled declaratively, thus users must restore the previous set of volumes, which will be read as a cancellation for the volume update and migration.
+
+A possible way to cancel the migration consists apply the old version of the VM declaration.
+
+## Volume migration failure and recovery
+
+If a volume migration fails, KubeVirt will continue to try to migrate until the volume update is cancelled. As a result, if the original failure was temporary, a subsequent migration could succeed; otherwise, KubeVirt continues to generate new migrations with exponential backoff time.
+
+To recover from a persistent failure, users must revert the volume set to its original state, indicating the cancellation of the volume migration.
+
+### Manual recovery required
+
+If for any reasons the VMI disappears, then the volume migration isn't retried anymore. This might happen if the users inadvertently shutdown the VM or the VMI is accidentally deleted.
+
+However, in these situations, the VM spec is in an inconsistent state because the volume set contains the destination volumes but the copy wasn't successful, and users could failed to boot correctly the VM. For this reason the VM is marked with the condition `ManualRecoveryRequired` and KubeVirt will refuses to start a VM which has this condition.
```

Comment:
```suggestion
However, in these situations, the VM spec is in an inconsistent state because the volume set contains the destination volumes but the copy wasn't successful, and users could fail to boot correctly the VM. For this reason the VM is marked with the condition `ManualRecoveryRequired` and KubeVirt will refuses to start a VM which has this condition.
```